src/pages/index.js

```diff
@@ -6,41 +6,73 @@ import ModalWithImage from "../components/ModalWithImage.js";
 import ModalWithForm from "../components/ModalWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import {
-  initialCards,
   config,
   profileEditButton,
   profileForm,
-  profileTitle,
-  profileDescription,
   profileTitleInput,
   profileDescriptionInput,
   cardList,
-  cardTemplate,
   addCardButton,
   cardForm,
-  cardTitleInput,
-  cardLinkInput,
+  editAvatarButton,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import ModalWithConfirmation from "../components/ModalWithConfirmation.js";
 
-// Instantiations
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createCard(cardData);
-      cardSection.addItem(cardElement);
-    },
+// User Info
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  descriptionSelector: ".profile__description",
+  avatarSelector: ".profile__avatar",
+});
+
+// API Calls
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "24d24150-f342-4b21-ba37-43c96c87f158",
+    "Content-Type": "application/json",
   },
-  ".cards__list"
-);
+});
 
-cardSection.renderItems();
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
+    userInfo.setUserInfo(userData);
+    userInfo.setUserAvatar(userData.avatar);
+    const cardSection = new Section(
+      {
+        items: cardData,
+        renderer: (cardData) => {
+          const card = new Card(
```

Comment:
It's highly recommended to make a separate function `createCard` for creating a card not to copy-paste the code

```js		

function createCard(item) {
    // here you create a card
  return cardElement.getCard()
}

```


So, here will be


```js
        renderer: (cardData) => {
          const card = createCard(cardData);
          cardSection.addItem(card);
        },

```